tm2/pkg/amino/amino.go

```diff
@@ -130,7 +130,7 @@ func UnmarshalAnySized(bz []byte, ptr interface{}) error {
 }
 
 func MarshalJSON(o interface{}) ([]byte, error) {
```

Comment:
Shouldn't we be consistent and update the top-level methods, too?

Reply:
It's not strictly necessary, and changing top-level functions would be much more impactful on the code base. In any case, we should probably just apply the smallest change first.